omod/src/main/resources/apps/mksreports_extension.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "id": "${project.parent.artifactId}.downloadPatientHistory",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "link",
+        "label": "mksreports.patienthistory.linklabel",
+        "url": "module/mksreports/patientHistory.form?patientId={{patient.patientId}}&download=true",
+        "icon": "icon-print",
+        "order": 14
```

Comment:
We need to add a `"requiredPrivilege"` attribute. The privilege name could be `"Can Download Patient History"`.

Reply:
requiredPrivilege added